client/src/Features/MicrosoftLearn/MicrosoftLearnFilterPane.tsx

```diff
@@ -0,0 +1,244 @@
+          setMainOpen(true);
+        }}
+        disabled={isLoadingCatalog ? true : false}
+      />
+
+      <Panel
+        headerText={getHeader()}
+        isOpen={panelIsOpen}
+        onDismiss={() => setPanelOpen(false)}
+        closeButtonAriaLabel="Close"
+        isFooterAtBottom={true}
+        className={classes.filterPanelTabView}
+        onRenderNavigationContent={backToMainFilters}
+        type={PanelType.smallFixedNear}
+      >
+        <>
+          {mainIsOpen ? (
+            <>
+              <ActionButton
+                onClick={() => setOpen(false, true, false, false, false)}
+                className={classes.mainPanelActionButtons}
+              >
+                <Text className="buttonTitle">Products</Text>
+                <Icon iconName="ChevronRight" />
+              </ActionButton>
+
+              <ActionButton
+                onClick={() => setOpen(false, false, true, false, false)}
+                className={classes.mainPanelActionButtons}
+              >
+                <Text className="buttonTitle">Roles</Text>
+                <Icon iconName="ChevronRight" />
+              </ActionButton>
+
+              <ActionButton
+                onClick={() => setOpen(false, false, false, false, true)}
+                className={classes.mainPanelActionButtons}
+              >
+                <Text className="buttonTitle">Levels</Text>
+                <Icon iconName="ChevronRight" />
+              </ActionButton>
+
+              <ActionButton
+                onClick={() => setOpen(false, false, false, true, false)}
+                className={classes.mainPanelActionButtons}
+              >
+                <Text className="buttonTitle">Types</Text>
+                <Icon iconName="ChevronRight" />
+              </ActionButton>
+            </>
+          ) : productIsOpen ? (
+            <FilterComponent type={FilterType.products} name='Products'/>                  
+          ) : roleIsOpen ? (
+            <FilterComponent type={FilterType.roles} name='Roles'/>
+          ) : levelIsOpen ? (
+            <FilterComponent type={FilterType.levels} name='Levels'/>
+          ) : (
+            <FilterComponent type={FilterType.types} name='Types'/>
+          )}
+        </>
+        {PanelFooterContent(filteredCatalogContent?.length)}
+      </Panel>
+    </>
+  );
+})
+};
+
+const FilterPaneInner = ({ styles }: IStylesOnly<FilterPaneStyles>): JSX.Element | null => {
+  const learnStore = useStore('microsoftLearnStore');
+  const [width, setWidth] = useState(window.innerWidth);
+  const classes = themedClassNames(styles);
+
+  useEffect(() => {
+    const debouncedHandleResize = debounce(function handleResize() {
+      setWidth(window.innerWidth);
+    }, 500);
+    window.addEventListener('resize', debouncedHandleResize);
+    return () => {
+      window.removeEventListener('resize', debouncedHandleResize);
+    };
+  });
+
+  return useObserver(() => {
+    return width > 768 ? (
```

Comment:
any specific reason to use `768` value? I'm seeing this at more than 1 place

Reply:
As far as I referred, 768px is the commonly used screen size for tabs. Using the same here to re-render as we re-size. Have added a `const` for the same to make this more readable.